src/ng/http.js

```diff
@@ -1249,7 +1249,11 @@ function $HttpProvider() {
           cache,
           cachedResp,
           reqHeaders = config.headers,
-          url = buildUrl(config.url, config.paramSerializer(config.params));
+          // origUrl could be a $sce trusted URL which used a wrapper class
+          origUrl = config.url,
+          url = $sce.valueOf(origUrl);
+
+      url = buildUrl(url, config.paramSerializer(config.params));
```

Comment:
I think it is simpler if all changes inside `$http` are done inside `buildUrl`. Something like this:

```diff
 function buildUrl(url, serializedParams) {
   if (serializedParams.length > 0) {
+    url = $sce.valueOf(url); 
     url += ((url.indexOf('?') === -1) ? '?' : '&') + serializedParams;
   }
   return url;
 }
```